cypress/support/config/services.js

```diff
@@ -1211,7 +1211,7 @@ const genServices = appEnv => ({
       },
       liveRadio: { path: undefined, smoke: false },
       mediaAssetPage: {
-        path: undefined,
+        path: isLive(appEnv) ? undefined : '/thai/thailand-23248713',
```

Comment:
this URL will stop working soon - posted on Slack here:

https://bbcnews.slack.com/archives/CJT0284G2/p1580233377036800

These TEST URLs currently work, but will start breaking soon (probably tomorrow)

- /japanese/video-23248670
- /korean/media-23248686
- /punjabi/media-23248705
- /sinhala/23248970
- /uzbek/sport-23248721
- /thai/thailand-23248713
- /news/av/business-23249447

There will be a near-future deployment that will make every MAP from any of these services work in TEST:

- Thai
- Punjabi
- Afaan Oromoo
- Amharic
- Tigrinya
- Marathi
- Telugu
- Gujarati

But there will be an interim time when only Pidgin, Igbo and Yoruba will route to Simorgh